src/olympia/ratings/forms.py

```diff
@@ -42,7 +43,8 @@ class RatingForm(RatingReplyForm):
     rating = forms.ChoiceField(
         zip(range(1, 6), range(1, 6)), label=_(u'Rating')
     )
-    flags = re.I | re.L | re.U | re.M
+    # re.L flag has been removed in py3.6 for text_type strings.
```

Comment:
Maybe make the comment even more explicit ? The docs say `Unicode matching is already enabled by default in Python 3 for Unicode (str) patterns, and it is able to handle different locales/languages` ; so it's not so much that it's been removed for text type strings, it's that it's enabled by default anyway. I was concerned before reading this that we would be removing something without providing a solution.